### PR TITLE
[ADD sortedIndex.md] 

### PR DESCRIPTION
**Pull request by kriadmin**
## Description

**Lodash[ADD]** #100 -> https://lodash.com/docs/4.17.4#sortedIndex 

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## [Lodash Backlog](https://github.com/Chalarangelo/30-seconds-of-code/issues/100)

- [x] I added the prefix [UPDATE: `method.md`] or [ADD: `method.md`]
- [x] I have referenced the `method` to the lodash backlog.

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/sortedIndex.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### sortedIndex
+
+Returns the lowest index at which value should be inserted into array in order to maintain its sort order
+
+```js
+const sortedIndex = (arr,n) => {
+    arr[0] > arr[1] ? (anarray = arr.reverse(),isReversed = true) : (anarray = arr,isReversed = false);
```
Comment (atomiks) [inline snippets/sortedIndex.md]: `anarray` and `isReversed` become global variables here

File: snippets/sortedIndex.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### sortedIndex
+
+Returns the lowest index at which value should be inserted into array in order to maintain its sort order
+
+```js
+const sortedIndex = (arr,n) => {
+    arr[0] > arr[1] ? (anarray = arr.reverse(),isReversed = true) : (anarray = arr,isReversed = false);  
+	val = anarray.findIndex( el => {
+	return n <= el
```
Comment (atomiks) [inline snippets/sortedIndex.md]: You can just do `.findIndex(el => n <= el);`

File: snippets/sortedIndex.md (hunk 1 above)
Comment (kriadmin) [reply snippets/sortedIndex.md]: Oh yeah let me fix it

**Comment by Chalarangelo** [comment]
@atomiks Had the same thoughts, I just fixed these problems! 😆

**Comment by Chalarangelo** [comment]
@atomiks I like your approach, it also seems a lot faster!

**Comment by kriadmin** [comment]
@atomiks @Chalarangelo Updated it.

**Pull request merged**